tests/unit/test_model_card.py

```diff
@@ -597,6 +622,26 @@ def fixture_model_overview_example():
     return test_example
 
 
+@pytest.fixture(name="model_package_example")
+def fixture_model_package_example():
+    """Example ModelPackage instance"""
+    test_example = ModelPackage(
+        model_package_arn=MODEL_PACKAGE_ARN,
+        model_package_description=MODEL_PACKAGE_DESCRIPTION,
+        model_package_group_name=MODEL_PACKAGE_GROUP_NAME,
+        model_package_name=MODEL_PACKAGE_NAME,
+        model_approval_status=MODEL_APPROVAL_STATUS,
+        model_package_status=MODEL_PACKAGE_STATUS,
+        model_package_version=MODEL_PACKAGE_VERSION,
+        approval_description=APPROVAL_DESCRIPTION,
+        domain=DOMAIN,
+        task=TASK,
+        created_by=CREATED_BY,
+        source_algorithms=SOURCE_ALGORITHMS,
+        inference_specification=INFERENCE_SPECIFICATION
+    )
+    return test_example
+
 @pytest.fixture(name="intended_uses_example")
 def fixture_fixture_intended_uses_example():
     """Example intended uses instance."""
```

Comment:
the actual unit test function is missing, like 
- `test_create_model_card_with_model_package` [example](https://github.com/aws/sagemaker-python-sdk/blob/master/tests/unit/test_model_card.py#L675)
- `test_model_package_autodiscovery` [example](https://github.com/aws/sagemaker-python-sdk/blob/master/tests/unit/test_model_card.py#L1000)

Also make sure all the ValueError could be potential raised should also be covered in the new tests [example](https://github.com/aws/sagemaker-python-sdk/blob/ce2fb7805af9fd304645aaf02c4564168346dbcd/tests/unit/test_model_card.py#L1024)

Reply:
Yup, I know about this test. But we cannot create a model card with model package yet because of the schema which is still in the pipeline, so did not add that. Will add those tests after adding that functionality.

Reply:
Added these tests as well.

Reply:
Cool! 

I think even though schema which is still in the pipeline we could still do that here though since this is unit test it is not gonna call external endpoint and the response is mocked through `@patch("sagemaker.Session")`.